RELEASE.md

```diff
@@ -0,0 +1,37 @@
+# Release Process
+
+Pack follows a 6 week release cadence, composed of 3 phases:
+  - [Development](#development)
+  - [Feature Complete](#feature-complete)
+  - [Release Finalization](#release-finalization)
+
+## Development
+Our development flow is detailed in [Development](DEVELOPMENT.md)
+
+## Feature Complete
+### Process
+5 business days prior to a scheduled release, we enter `feature complete`. A release branch (in the form `release/<VERSION>`) is created, and used for `UAT` (User Acceptance testing).
+
+During this period, relevant changes may be merged into the release branch, based on assessment by the [CCB](#ccb) of the impact, effort and risk of including the changes. Any other change may get merged into `main` through the normal process, and will make it into the next release.
```

Comment:
```suggestion
During this period, relevant changes may be merged into the release branch, based on assessment by the [CCB](#change-control-board) of the impact, effort and risk of including the changes. Any other change may get merged into `main` through the normal process, and will make it into the next release.
```

Reply:
Also, I wonder if the mention of CCB is really necessary or if we should just inline ~~"CCB"~~ -> "pack maintainers".

Reply:
Agreed, I just removed mention of CCB